src/components/Chart/LineChart.jsx

```diff
@@ -238,7 +386,29 @@ LineChart.defaultProps = {
   showAxis: false,
   tickPadding: 8,
   tickSizeOuter: 0,
-  tickSizeInner: 5
+  tickSizeInner: 5,
+  pointRadius: 4,
+  pointFillColor: 'black',
+  pointStrokeWidth: 10,
+  pointStrokeColor: 'rgba(0, 0, 0, 0.3)'
+}
+
+class Tooltip extends Component {
+  render() {
+    const { children, x } = this.props
+
+    return (
+      <div
+        ref={node => (this.node = node)}
+        className={styles.LineChartTooltip}
+        style={{
+          left: x + 'px'
```

Comment:
A `transform: translateX` would not trigger a relayout here.

https://csstriggers.com/transform

> left
Changing left alters the geometry of the element. That means that it may affect the position or size of other elements on the page, both of which require the browser to perform layout operations.
Once those layout operations have completed any damaged pixels will need to be painted and the page must then be composited together.

> transform
Changing transform does not trigger any geometry changes or painting, which is very good. This means that the operation can likely be carried out by the compositor thread with the help of the GPU.